### ISSUE TITLE
Improve accessibility

### ISSUE THREAD

**Issue by Snugug** [elk-zone/elk#92]
* [ ] `zen-hide` sidebar should be an `aside` instead of a `div`
  * [ ] "A mastodon client made with <3" and "las build" should be in `p` tags instead of `div`s
  * [ ] Username should be a header or a `p` instead of a `div`
  * [ ] Profile image should be wrapped in the same tag as the username stuff, and should be a `button` because it's not pointing to a separate URL but rather toggles a flyout menu (it's should semantically be the same as a "hamburger" flyout toggle). In fact, the `button` should probably be lifted up to the `div` that contains the button styling
    * [ ] Speaking of the account flyout menu, that should capture tab focus, include a "close" button, and should close when a user hits `esc`
  * [ ] The light grey used throughout the dark theme isn't [accessible for the sizes it's used](https://contrast-ratio.com/#rgba%28255%2C255%2C255%2C.35%29-on-rgb%2817%2C17%2C17) at (it looks like white with .35 opacity applied; I'd generally recommend against applying opacity to text because it gets hard to use DevTools to determine color contrast)`
* [ ] The glass effect that holds small-screen nav and headers (at least in dark mode) doesn't allow for enough color contrast when over light colors (especially white). I'd generally recommend (as a quick test, because these are mostly white and black, and the orange is close to white in lightness) determining color contrast by throwing a white background behind the effect and doing a contrast ratio test against the resultant rendered background color
* [ ] The account sidebar and nav shouldn't require a user to hover to see the ability to enable/view. Instead, a persistent button to show/hide these items should be made available.
